requirements.txt

```diff
@@ -31,10 +33,11 @@ django-extensions==2.2.9
 django-heroku==0.3.1
 django-model-utils==4.0.0
 django-redis==4.11.0
+django-rest-swagger==2.2.0
 django-storages==1.9.1
 django-stubs==1.5.0
-djangorestframework==3.11.0
```

Comment:
You don't want to remove djangorestframework from your requirements file. You need that. The app still works because pip will have installed it as a dependency of drf-yasg.

I think you intended to remove django-rest-swagger==2.2.0 when you swapped to using drf-yasg.

Reply:
You are correct, thank you!
I have fixed it. [Commit](https://github.com/civictechindex/CTI-website-backend/pull/3/commits/d4e7b8edf9c1a4521b2d1338f9e276fdf4f733de)